_posts/2017-05-04-install-party.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: post
+title: "Install Party + Social"
+slug: install-party
+text: 'tomesh monthly install party and social'
+location: Semaphore Studio 307, Claude T. Bissell, 140 St. George St  
+locationLink: http://osm.org/go/ZX6Bw~WNh--?m=
+date: 2017-05-04
+startTime: '19:00'
+endTime: '20:30'
+---
+
+Join us at {{page.startTime | date:"%-I:%M%P "}} on {{ page.date | date: '%B %d, %Y' }} for our monthly install party and working groups' check-in. This month we will be meeting at [{{ page.location }}]({{ page.locationLink }}) for a hands-on session before heading to a nearby bar for drinks.
```

Comment:
I'd remove the "working groups'" part? As I think this is more about installing stuff?